indico/modules/events/timetable/client/js/reducers.ts

```diff
@@ -407,12 +407,17 @@ export default {
         return state;
     }
   },
-  navigation: (state = {numDays: 2, offset: 0}, action) => {
+  navigation: (
+    state = {numDays: 2, offset: 0, currentDay: moment().format('YYYYMMDD')},
```

Comment:
This might not be a valid day for the event. I'd keep it as null initially and set a valid day at the same time we dispatch `setTimetableData` with the initial data.